### ISSUE TITLE
Files - open local downloaded file

### ISSUE THREAD

**Issue by licaon-kter** [yaxim-org/yaxim#267]
...not a browser to the URL.

Since you are showing previews of images, you already downloaded them, right?

**Comment by ge0rg** [comment]
Yes, but I'm using the com.koushikdutta.urlimageviewhelper library, which automatically downloads and caches all URLs that I throw at it. Unfortunately, it will also download all non-image files, and there is no easy way to access the local copies.

I'm very unlucky with it, but implementing it on my own is some work for which I don't have the time now. However, it would also enable a gallery etc., so it's on my TODO list.
